### DISCUSSION TITLE
Processing time

### DISCUSSION THREAD

**Discussion by BANadim** [ROBelgium/MSNoise#346] [Q&A] [answered]
Hello,

I am a new user of MSNoise I have installed it successfully and tried to run it. unfortunately it has been 3 days and there seems to be no update in the dashboard. Is this normal? I have attached images for reference.

![MSnoiseDashboard](https://github.com/ROBelgium/MSNoise/assets/164434688/7eb9d24a-093e-4b91-bdeb-50002c03bc86)

![info_j](https://github.com/ROBelgium/MSNoise/assets/164434688/e4757ecb-2d26-463e-8d35-2bf55e1325f4)

I thank you all for your patience I am still new to MSNoise.

**Comment by ThomasLecocq** [comment]
What command did you run to compute the CC?

**Reply by BANadim** [reply]
I used $ msnoise compute_cc.

![compute cc](https://github.com/ROBelgium/MSNoise/assets/164434688/e0aeb5d0-b92a-4558-8840-9444144b1a60)

**Reply by ThomasLecocq** [reply, marked as answer]
Well, the message is quite clear: you didn't define any filter in the configuration, so the compute_cc step cannot work, and the process is exiting. Using the web admin, go to the filter configuration, define at least one active filter and re-run the command.

**Comment by BANadim** [comment]
Hello,
Got it! thanks for your patience!



![compute CC](https://github.com/ROBelgium/MSNoise/assets/164434688/f7582638-01bf-4fb1-b494-e3bf094897bc)
